### ISSUE TITLE
The "time" command no longer works in autograding

### ISSUE THREAD

**Issue by emaicus** [Submitty/Submitty#5984]
At present, the `time` command fails silently when autograding despite being whitelisted.

**Comment by satishhhh** [comment]
i want to work on this issue

**Comment by satishhhh** [comment]
@emaicus  can you elaborate more on this issue which time-submission time,gradetime etc which one?

**Comment by bmcutler** [comment]
Actually, it does work now (as of Fall 21)

This example should work (hopefully I didn't introduce typos in modifying from more complex working example).  Maybe we were missing the RLIMIT_NPROC (might not need to be as high as 40 for a single threaded C++ program a.out).  Or maybe there was a problem with the system calls that has been fixed.  Or maybe it was the escaped characters

{
            "title" : "TEST WITH TIME",
            "details" : "./a.out arg1 arg2",
            "command" : "/usr/bin/time -f %e\\ elapsed\\ seconds ./a.out arg1 arg2",
            "resource_limits" : { "RLIMIT_CPU" : 10,
                                  "RLIMIT_NPROC" : 40 },
            "points" : 2,
            "validation" : [
                { "description" : "run time", "method" : "warnIfEmpty", "actual_file" : "STDERR.txt", /*RUNTIME*/ "show_actual" : "always", "deduction" : 0.0 },
                { "method" : "warnIfNotEmpty", "actual_file" : "STDOUT.txt", "show_actual" : "on_failure", "deduction" : 0.0 },
            ]
}

@poorna2152 if this works for you too -- then let's close the issue.

**Comment by poorna2152** [comment]
@bmcutler Yes this does work for me when running an autograding job in the jailed sandbox. However when running in the docker container the command no longer works. May be that is the issue mentioned here.

**Comment by bmcutler** [comment]
I did not use a docker container for that assignment -- for other reasons.  But yes, let's figure out what's different and why it doesn't work in a container.
